src/controller/CHIPDeviceController.cpp

```diff
@@ -1274,27 +1274,28 @@ CHIP_ERROR DeviceCommissioner::SendOperationalCertificate(DeviceProxy * device,
     return CHIP_NO_ERROR;
 }
 
-CHIP_ERROR DeviceCommissioner::ConvertFromNodeOperationalCertStatus(uint8_t err)
+CHIP_ERROR DeviceCommissioner::ConvertFromOperationalCertStatus(OperationalCredentials::OperationalCertStatus err)
 {
+    using OperationalCredentials::OperationalCertStatus;
     switch (err)
     {
-    case EMBER_ZCL_NODE_OPERATIONAL_CERT_STATUS_SUCCESS:
+    case OperationalCertStatus::kSuccess:
         return CHIP_NO_ERROR;
-    case EMBER_ZCL_NODE_OPERATIONAL_CERT_STATUS_INVALID_PUBLIC_KEY:
+    case OperationalCertStatus::kInvalidPublicKey:
         return CHIP_ERROR_INVALID_PUBLIC_KEY;
-    case EMBER_ZCL_NODE_OPERATIONAL_CERT_STATUS_INVALID_NODE_OP_ID:
+    case OperationalCertStatus::kInvalidNodeOpId:
         return CHIP_ERROR_WRONG_NODE_ID;
-    case EMBER_ZCL_NODE_OPERATIONAL_CERT_STATUS_INVALID_NOC:
+    case OperationalCertStatus::kInvalidNOC:
         return CHIP_ERROR_CERT_LOAD_FAILED;
```

Comment:
Invalid NOC is not a loading failure, but a bad value. Perhaps "CHIP_ERROR_UNSUPPORTED_CERT_FORMAT"

Reply:
I think "CERT_LOAD" means "loading it into a certificate store".

But yes, on the server CHIP_ERROR_UNSUPPORTED_CERT_FORMAT is one of many error codes (including CHIP_ERROR_CERT_LOAD_FAILED) which get mapped to kInvalidNOC.  I can change this line to use that one instead of the one it uses now.